src/main/java/com/example/universe/simulator/entityservice/services/GalaxyService.java

```diff
@@ -45,6 +51,7 @@ public Galaxy add(Galaxy entity) throws AppException {
     }
 
     @Transactional
+    @CachePut(key = "#p0.id")
```

Comment:
I think using `entity.id` instead of `#p0.id` makes code more readable.

Reply:
Modified. @CachePut now uses conditional.